.circleci/config.yml

```diff
@@ -1,23 +1,18 @@
-defaults: &defaults
-  working_directory: ~/app
-  environment:
-    JAVA_TOOL_OPTIONS: -Xmx1024m -XX:ConcGCThreads=2 -XX:ParallelGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2
-
 version: 2.1
 
 orbs:
+  hmpps: ministryofjustice/hmpps@1.0.0
   owasp: entur/owasp@0.0.6
 
+defaults: &defaults
```

Comment:
I guess this defaults parameter is just a placeholder now and is expected to be empty?